### ISSUE TITLE
Allow album selection using the keyboard

### ISSUE THREAD

**Issue by michaelkpfeifer** [michaelkpfeifer/roon-echo#513]
It should be possible to select an album using the keyboard. This is something ESLint already complains about. Search for "jsy-a11y" in `AlbumCard.tsx` .
